pkg/services/oracle/neofs/neofs.go

```diff
@@ -96,15 +93,15 @@ func parseNeoFSURL(u *url.URL) (*object.Address, []string, error) {
 	return objectAddr, ps[2:], nil
 }
 
-func getPayload(ctx context.Context, c *client.Client, addr *object.Address) ([]byte, error) {
+func getPayload(ctx context.Context, c client.Client, addr *object.Address) ([]byte, error) {
```

Comment:
IIRC `client.Client` is an interface?

Reply:
it was not Interface in `v1.24.0` but in `v1.27.0` it is

Reply:
Good, then it all makes sense.